### ISSUE TITLE
Updated analysis: transcriptomic dimension reduction plots

### ISSUE THREAD

**Issue by kgaonkar6** [AlexsLemonade/OpenPBTA-analysis#872]
#### What analysis module should be updated and why?
While updating transcriptomic-dimension-reduction module for v18 release @jharenza noticed a `NA` value in the plots. For example https://github.com/kgaonkar6/OpenPBTA-analysis/blob/tsne-base/analyses/transcriptomic-dimension-reduction/plots/rsem_stranded_log_broad_histology.pdf

I believe the NA arises from a manual color palette given in the code which has 21 colors 
https://github.com/kgaonkar6/OpenPBTA-analysis/blob/37b46f6b443a449a78b6f4c5394fd67e24e2d59e/analyses/transcriptomic-dimension-reduction/util/dimension-reduction-functions.R#L264-L271

But from the new base histology we have 22 broad_histology:
```r
> check$broad_histology %>% unique()
 [1] "Low-grade astrocytic tumor"                    "Tumors of sellar region"                      
 [3] "Mesenchymal non-meningothelial tumor"          "Diffuse astrocytic and oligodendroglial tumor"
 [5] "Benign tumor"                                  "Ependymal tumor"                              
 [7] "Embryonal tumor"                               "Tumor of cranial and paraspinal nerves"       
 [9] "Histiocytic tumors"                            "Choroid plexus tumor"                         
[11] "Meningioma"                                    "Neuronal and mixed neuronal-glial tumor"      
[13] "Lymphoma"                                      "Metastatic tumors"                            
[15] "Non-tumor"                                     "Germ cell tumor"                              
[17] "Other astrocytic tumor"                        "Tumor of pineal region"                       
[19] "Histiocytic tumor"                             "Melanocytic tumor"                            
[21] "Pre-cancerous lesion"                          "Non-CNS tumor"   
```

#### What changes need to be made? Please provide enough detail for another participant to make the update.
This needs to be updated to 22 distinct colors:
https://github.com/kgaonkar6/OpenPBTA-analysis/blob/37b46f6b443a449a78b6f4c5394fd67e24e2d59e/analyses/transcriptomic-dimension-reduction/util/dimension-reduction-functions.R#L264-L271

#### What input data should be used? Which data were used in the version being updated?
pbta-gene-expression-rsem-fpkm.polya.rds
pbta-gene-expression-rsem-fpkm.stranded.rds
pbta-gene-expression-kallisto.polya.rds
pbta-gene-expression-kallisto.stranded.rds

**Comment by jaclyn-taroni** [comment]
In a sense, this was resolved when #868 got merged because to pass CI we needed to add something to the palette. However, the problem tracked in #889 means we needed to add more colors to the palette because `broad_histology` has more values than broad histologies. That resulted in a 26 color palette, rather than a 22 color palette, being added in #868. I don't think there's much harm in including extra colors, but the results and plots in `master` for this module are affected by #889.
